chain/network/src/accounts_data/mod.rs

```diff
@@ -147,6 +158,19 @@ impl Cache {
         }
     }
 
+    pub fn is_tier1(&self, account_id: &AccountId) -> bool {
+        self.inner.read().account_data(account_id).is_some()
+    }
+
+    pub fn get_peers(&self, account_id: &AccountId) -> Vec<PeerId> {
+        let inner = self.inner.read();
+        let data = match inner.account_data(account_id) {
+            Some(it) => it,
+            None => return Vec::new(),
+        };
+        data.peers.iter().flat_map(|it| it.peer_id.clone()).collect()
```

Comment:
peer_id is optional (an additional constraint), it is the IP that's important.

(IP,Some(peer_id)) means: "to route a message to account_id, send it to peer_id at address <IP>"

(IP,None) means: "to route a message to account_id send it to node at address <IP> (no matter what peer_id it will present).